### PR TITLE
Updates to About Sui including copyedit, formatting, and style

### PR DESCRIPTION
**Pull request by randall-Mysten**
Edit pass for style, format, and clarity.

### CODE REVIEW

File: doc/src/learn/about-sui.md [hunk 1]
```diff
@@ -6,31 +6,33 @@ Sui is the first permissionless Layer 1 blockchain designed from the ground up t
...
 
 Sui is backed by a number of state-of-the-art [peer-reviewed works](../contribute/research-papers.md) and years of open source development.
 
 ### Transactions
 
-A transaction in Sui is a change to the blockchain. This may be a *simple transaction* affecting only single-owner, single-address objects, such as minting an NFT or transferring it or another token. These transactions may bypass the consensus protocol in Sui.
+A transaction in Sui is a change to the blockchain. This may be a *simple transaction* affecting only single-owner, single-address objects, such as minting an NFT or transferring an NFT to a different token. These *simple transactions* may bypass the consensus protocol in Sui.
```
Comment (randall-Mysten) [inline doc/src/learn/about-sui.md:23]: @Clay-Mysten I'm not sure whether this was intentional to say "...or transferring it or another token"

File: doc/src/learn/about-sui.md [hunk 2]
```diff
@@ -55,7 +57,7 @@ By design, Sui validators (nodes) can effectively scale the network throughput i
...
 
-This experiment uses a configuration where each client submits a batch of 100 transactions (i.e., transfers to 100 distinct recipients) with a single signature. This configuration captures the anticipated usage pattern of a highly scalable blockchain--e.g., a custodial wallet or game server operating at scale will likely need to submit hundreds or thousands of on-chain transactions per second. With a batch size of 1, a validator running on the same machine can process 20,000 TPS with 8 cores, and exhibits the same linear growth in throughput as more cores are added.
+This experiment uses a configuration where each client submits a batch of 100 transactions (such as transfers to 100 distinct recipients) with a single signature. This configuration captures the anticipated usage pattern of a highly scalable blockchain--for example, a custodial wallet or game server operating at scale will likely need to submit hundreds or thousands of on-chain transactions per second. With a batch size of 1, a validator running on the same machine can process 20,000 TPS with 8 cores, and exhibits the same linear growth in throughput as more cores are added.
```
Comment (randall-Mysten) [inline doc/src/learn/about-sui.md:60]: removed the latin abbreviations for localization

File: doc/src/learn/about-sui.md [hunk 3]
```diff
@@ -6,31 +6,33 @@ Sui is the first permissionless Layer 1 blockchain designed from the ground up t
 
 ## What Sui is
 
-Sui is a smart contract platform maintained by a permissionless set of validators that play a role similar to validators or miners in other blockchain systems.
+Sui is a [smart contract](sui-glossary#smart-contract) platform maintained by a permissionless set of [validators](sui-glossary#validator) that play a role similar to validators or miners in other blockchain systems.
```
Comment (Clay-Mysten) [inline doc/src/learn/about-sui.md:9]: I don't think these links will work. You need to Markdown file extension, for sure.

**Review by Clay-Mysten — CHANGES_REQUESTED** [review]
Thanks, Randall. I made some small edits inline in a distinct commit here. Once the links are fixed, this should be good to go. Nice work!

File: doc/src/learn/about-sui.md (hunk 3 above)
Comment (randall-Mysten) [reply doc/src/learn/about-sui.md:9]: Thanks, I will confirm. I see this link in the same format working on the site:
https://github.com/MystenLabs/sui/blob/main/doc/src/learn/sui-glossary.md?plain=1#L121

But I also see a link to the same using .md:
https://github.com/MystenLabs/sui/blob/main/doc/src/learn/sui-glossary.md?plain=1#L19

Both work in public docs

Comment (Clay-Mysten) [reply doc/src/learn/about-sui.md:9]: > Thanks, I will confirm. I see this link in the same format working on the site: https://github.com/MystenLabs/sui/blob/main/doc/src/learn/sui-glossary.md?plain=1#L121
> 
> But I also see a link to the same using .md: https://github.com/MystenLabs/sui/blob/main/doc/src/learn/sui-glossary.md?plain=1#L19
> 
> Both work in public docs

Yes, these will work on docs.sui.io but not in the GitHub Markdown version. I will fix the FAQ. Thanks for the catch!

Comment (randall-Mysten) [reply doc/src/learn/about-sui.md]: ah, thanks for the clarification. I can add the glossary file to this PR

**Review by Clay-Mysten — APPROVED** [review]
Thanks, Randall!

**Pull request merged**